### PR TITLE
Made toy mechs remote controllable

### PR DESCRIPTION
**Pull request by Cruix**
:cl:
rscadd: The toy mechs from arcade machines are now remote controllable.
/:cl:

The gift shop starts with two random toy mechs and two toy mech controllers. The mechs can fight, and certain mechs have special abilities.

mechs
mecha controller

### CODE REVIEW

File: code/__HELPERS/lists.dm [hunk 1]
```diff
@@ -28,7 +28,7 @@
 			output += "[input[index]][comma_text]"
 			index++
 
-		return "[output][and_text][input[index]]"
+		return "[output][comma_text][and_text][input[index]]"
```
Comment (Cruix) [inline code/__HELPERS/lists.dm:31]: Also Oxford comma.

**Comment by Super3222** [comment]
I'm not gunna lie, this is pretty sick.

**Comment by ShadowDeath6** [comment]
holy fuck

**Comment by ghost** [comment]
wait do the projectiles actually kill people because i'm seeing 30 force on a lot of these things and i'm worried

**Comment by Cruix** [comment]
The projectiles delete themselves and do nothing if they hit anything that is not a toy mech.

And no we are not letting people emag these to make the projectiles real.

**Comment by ShadowDeath6** [comment]
PR of the year tag

**Comment by Kmc2000** [comment]
Holy shit

Can we bypass the feature freeze please...

**Comment by X-TheDark** [comment]
I'm fine with this being merged, it's interesting enough to merit that respect.

**Comment by Super3222** [comment]
This is why we have a Clerk

**Comment by ShadowDeath6** [comment]
merg pls

**Comment by X-TheDark** [comment]
Merged but I don't think I'll be able to update the server, due to me leaving.

**Comment by ShadowDeath6** [comment]
praise

**Pull request merged**